alf/environments/suite_carla.py

```diff
@@ -1023,10 +1035,13 @@ def get_current_time_step(self, current_frame):
                                                 self._actor.get_location())
         v = self._actor.get_velocity()
         obs['speed'] = np.float32(
-            np.float32(v.x * v.x + v.y * v.y + v.z * v.z))
+            np.sqrt(np.float32(v.x * v.x + v.y * v.y + v.z * v.z)))
```

Comment:
Is this a serious issue? It seems that in principle before and after are equivalent.

Reply:
The range would huge without the sqrt. I thought it shouldn't affect the training much. But somehow the training is better after fixing this. Although one training is not enough to draw the conclusion.